Main/Source/iconf.cpp

```diff
@@ -666,7 +666,7 @@ void ivanconfig::ShowItemsAtPlayerSquareChanger(cycleoption* O, long What)
 
 void ivanconfig::WindowWidthChanger(numberoption* O, long What)
 {
-  if(What < 800) What = 800;
+  if(What < 480) What = 480;
```

Comment:
You swap the width and height...

Reply:
ugh.. I missed this comment.. I guess github could highlight the news on the page we are reading? may be suggest at github forums :)